extensions/wikia/CommunityPage/tests/LinkHelperTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+class LinkHelperTest extends WikiaBaseTest {
+	const ANON = false;
+	const LOGGED_IN = true;
+	const ANNON_EDITS_ALLOWED = false;
+	const ANNOON_EDITS_FORBIDDEN = true;
+
+	const ARTICLE_NAME = 'test_article';
+	const ARTiCLE_LOCAL_URL = '/wiki/test_article';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant "ARTiCLE_LOCAL_URL" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS115)